pkgs/servers/monitoring/prometheus/gitlab-ci-pipelines-exporter.nix

```diff
@@ -2,23 +2,29 @@
 
 buildGoModule rec {
   pname = "gitlab-ci-pipelines-exporter";
-  version = "0.4.9";
+  version = "0.5.3";
 
   src = fetchFromGitHub {
     owner = "mvisonneau";
     repo = pname;
     rev = "v${version}";
-    sha256 = "13zs8140n4z56i0xkl6jvvmwy80l07dxyb23wxzd5avbdm8knypz";
+    sha256 = "1i0ry93w2pk8m7i4fbdn1xvsmvjn83d6lr6l7yywid9d0kmkjz13";
   };
 
-  vendorSha256 = "1k620r3d1swhj7cfmqjh5n08da2a6w87fwrsajl0y324iyw2chsa";
+  subPackages = [ "cmd/${pname}" ];
 
+  ldflags = [
+    "-X main.version=v${version}"
+  ];
+
+  vendorSha256 = "UszZaxql5sO4DSDVD/UIjeeDojCQEPF+ATL7O9MaTG8=";
   doCheck = true;
 
   meta = with lib; {
     description = "Prometheus / OpenMetrics exporter for GitLab CI pipelines insights";
     homepage = "https://github.com/mvisonneau/gitlab-ci-pipelines-exporter";
     license = licenses.asl20;
+    platforms = platforms.unix ++ platforms.darwin;
```

Comment:
For `buildGoModule`, this should only be set if the package doesn't work with the default platforms (`platforms.linux ++ platforms.darwin`).

https://github.com/NixOS/nixpkgs/blob/ebaf33dd1ff8fc67b59629788f74d94b82db5442/pkgs/development/go-modules/generic/default.nix#L264-L267

```suggestion
```

Reply:
oh, very good to know thanks!